miio/gateway.py

```diff
@@ -643,74 +643,132 @@ def set_default_music(self):
 
 
 class GatewayLight(GatewayDevice):
-    """Light controls for the gateway."""
+    """
+    Light controls for the gateway.
+
+    The gateway LEDs can be controlled using 'rgb' or 'night_light' methods.
+    The 'night_light' methods control the same LEDs as the 'rgb' methods, but has a seprate memory for brightness and color.
+    Changing the 'rgb' light does not effect the stored state of the 'night_light', while changing the 'night_light' does effect the state of the 'rgb' light.
+    """
 
     @command()
-    def get_night_light_rgb(self):
-        """Unknown."""
-        # Returns 0 when light is off?"""
-        # looks like this is the same as get_rgb
-        # id': 65064, 'method': 'set_night_light_rgb', 'params': [419407616]}
-        # {'method': 'props', 'params':
-        # {'light': 'on', 'from.light': '4,,,'}, 'id': 88457} ?!
-        return self._gateway.send("get_night_light_rgb")
+    def rgb_status(self):
+        """
+        Get current status of the light.
+        Always represents the current status of the LEDs as oposed to 'night_light_status'.
+
+        Example:
+           {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)}
+        """
+        # Returns {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)} when light is off
+        state_int = self._gateway.send("get_rgb").pop()
+        brightness = int_to_brightness(state_int)
+        rgb = int_to_rgb(state_int)
+        is_on = brightness > 0
+
+        return {"is_on": is_on, "brightness": brightness, "rgb": rgb}
+
+    @command()
+    def night_light_status(self):
+        """
+        Get status of the night light.
+        This command only gives the correct status of the LEDs if the last command was a 'night_light' command and not a 'rgb' light command, otherwise it gives the stored values of the 'night_light'.
+
+        Example:
+           {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)}
+        """
+        state_int = self._gateway.send("get_night_light_rgb").pop()
+        brightness = int_to_brightness(state_int)
+        rgb = int_to_rgb(state_int)
+        is_on = brightness > 0
+
+        return {"is_on": is_on, "brightness": brightness, "rgb": rgb}
+
+    @command(
+        click.argument("brightness", type=int),
+        click.argument("rgb", type=(int, int, int)),
+    )
+    def set_rgb(self, brightness: int, rgb: (int, int, int)):
```

Comment:
```suggestion
    def set_rgb(self, brightness: int, rgb: Tuple[int, int, int]):
```

Also, you need to import `Tuple` from the typing module for this.